### ISSUE TITLE
APPS `do_X509_sign()`: is it complete w.r.t. RFC 5280 compliance?

### ISSUE THREAD

**Issue by DDvO** [openssl/openssl#19805]
Meanwhile, `do_X509_sign()` makes sure that, by default, subject key identifier (SKID) and issuer key identifier (AKID) are added as far as required by RFC 5280. I also makes sure that X.509 version 3 is set whenever X.509 extensions are included.

Is there anything else the function should do for ensuring RFC 5280 compliance for new certs by default?

**Comment by DDvO** [comment]
This is a spin-off from #16006.

**Comment by t8m** [comment]
I do not think this issue is really actionable as we do not aim for 100% RFC 5280 compliance. Closing.
